en-US/main.lang

```diff
@@ -737,3 +737,33 @@ See what’s new with Firefox
 # Description for Open Graph sharing in new What's new pages
 ;Faster, more responsive and powerfully private. Check out the latest from the browser that has your back.
 Faster, more responsive and powerfully private. Check out the latest from the browser that has your back.
+
+
+# Used on Privacy Notice landing page
+;Show More
+Show More
+
+
+# Used on Privacy Notice landing page
+;Show Less
+Show Less
+
+
+# Used on Privacy Notice landing page
+;Choose how you want to share this data in Firefox.
+Choose how you want to share this data in Firefox.
+
+
+# Dropdown menu to preference options
```

Comment:
This is completely off… The button opens Firefox preferences, in the Data section, to select your option.